crates/wash-cli/src/app/mod.rs

```diff
@@ -18,7 +18,7 @@ use crate::appearance::spinner::Spinner;
 mod output;
 
 #[derive(Debug, Clone, Subcommand)]
-pub(crate) enum AppCliCommand {
+pub enum AppCliCommand {
```

Comment:
Help me understand all these `pub(crate)` -> `pub` changes?

Reply:
That was required to have `wash` be a binary in `src/bin` and `wash-cli` being just a library.

The `pub(crate)` is redundant here anyway